protocol/communities/community.go

```diff
@@ -31,6 +31,8 @@ import (
 
 const signatureLength = 65
 
+var GrantExpirationTime = 24 * time.Hour
```

Comment:
```suggestion
var GrantExpirationTime = 72 * time.Hour
```
I remembered that we aim to avoid having clients with unconfirmed memberships in case the control node goes offline over a weekend. We should double check it with @John-44 though.